<launch>

<!-- joy_node -->
<node respawn="true" pkg="joy" type="joy_node" name="joy">
  <param name="dev" type="string" value="/dev/input/js0" />
  <param name="deadzone" value="0.12" />
  <param name="autorepeat_rate" value="25" />
</node>

<!-- teleop_twist_joy -->
<node pkg="teleop_twist_joy"   type="teleop_twist_joy_node"  name="teleop_twist_joy" >
  <rosparam command="load" ns="/teleop_twist_joy" file="$(find teleop_twist_joy)/config/ps2.yaml" />
  <rosparam command="load" ns="/teleop_twist" file="$(find teleop_twist_joy)/config/robot.yaml" />
</node>

</launch>
